dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/persistence/file/advanced/FeatureFileOrchestrator.kt

```diff
@@ -64,7 +64,7 @@ internal class FeatureFileOrchestrator(
     )
 
     companion object {
-        private const val BASE_DIR_NAME_REG_EX = "([a-z]+[-|_])+"
+        private const val BASE_DIR_NAME_REG_EX = "([a-z]+(-|_))+"
```

Comment:
Do we have a list of all the available names so far? Why are we including the `_` character?

Reply:
the `_` is used in the NDK folders for some reason...I see it was not aligned with the new folders name format. For now all the other features are using the `-` separator. That's the list so far:
```
logs-pending-v2
logs-v2
ndk_crash_reports_intermediary_v2
ndk_crash_reports_v2
rum-pending-v2
rum-v2
session-replay-pending-v2
session-replay-v2
tracing-pending-v2
tracing-v2
```

Reply:
NDK feature doesn't use storage for batches, so we may exclude folders belonging to it, it is outside of the scope for file orchestrator

Reply:
ok that's great...and the way I see it neither the uploader is using it as the ndk_crash_file is just used to write the equivalent RUM event in their (pending|granted) folder at runtime ? Can you confirm this @0xnm ...this way I could remove that extra rule from there

Reply:
so it seems you can simplify that regex to `([a-z]+-)+`

Reply:
yes, NDK folders are out of scope of the upload pipeline

Reply:
@xgouchet yup doing this now after the confirmation ;)